arbitrator/prover/src/machine.rs

```diff
@@ -972,6 +1027,7 @@ pub fn get_empty_preimage_resolver() -> PreimageResolver {
 
 impl Machine {
     pub const MAX_STEPS: u64 = 1 << 43;
+    pub const NO_STACK_HASH: Bytes32 = Bytes32::new([255_u8; 32]);
```

Comment:
I don't think you should need that.
Something like 'Bytes32([255_u8;32])' and/or  something like '[255_u8;32].into()' should work

Reply:
`Bytes32([255_u8;32])` works!

Tried `.into()/from()` before but that complained about non-static fn used in static initialization.